app/views/users/show.html.erb

```diff
@@ -1 +1,23 @@
-<h1>Showing user</h1>
\ No newline at end of file
+<%= render 'shared/user_profile', user: @user %>
+
+<div class="details">
+
+    <div class="border_solid">
+        <h1>Bio</h1>
+        <p>
+        <%= @user.bio %>
+        </p>
+    </div>
+<% @user.three_recent_posts.each do |post| %>
+    <div class="border_solid" >
+        <h1>post <%= post.title %></h1>
+        <p>
+        <%= post.text %>
+        </p>
+        <p class="likes_box">Comments: <%= post.comments_counter %> Likes: <%= post.likes_counter %></p>
+    </div>
```

Comment:
- [ ] The part of the post on the user show page should be a link that takes the user to the post. Kindly do that.